charts/karpenter/values.yaml

```
@@ -134,6 +134,9 @@ logLevel: info
 # -- Cluster name.
 clusterName: ""
 
+# -- Minimum kubernetes version that will be supported
+k8sMinVersion: "1.19.0-0"
```

Is the -0 suffix required? 
```suggestion
kubernetesMinVersion: "1.19.0-0"
```

---

I am not sure. but that's the format used by knative.